lib/tier-utils.js

```diff
@@ -88,23 +88,27 @@ export const getCollectiveContributionCardsOrder = collective => {
   return get(collective.settings, TIERS_ORDER_KEY, []);
 };
 
-export const sortTiers = (baseTiers, orderKeys, hasCustomContribution) => {
+export const sortTiers = (baseTiers, orderKeys, hasCustomContribution, hasCryptoContribution) => {
   const tiers = baseTiers.filter(tier => tier.type !== TierTypes.TICKET);
   if (hasCustomContribution) {
     tiers.push('custom');
   }
+  if (hasCryptoContribution) {
+    tiers.push('crypto');
+  }
 
   return orderBy(tiers, tier => {
-    const itemKey = tier === 'custom' ? 'custom' : tier.id;
+    const itemKey = tier === 'custom' || tier === 'crypto' ? tier : tier.id;
     const index = orderKeys.findIndex(key => key === itemKey);
     return index === -1 ? Infinity : index; // put unsorted tiers at the end
   });
 };
 
 export const sortTiersForCollective = (collective, baseTiers) => {
   const orderKeys = getCollectiveContributionCardsOrder(collective);
-  const hasCustomContribution = !collective.settings?.disableCustomContributions;
-  return sortTiers(baseTiers, orderKeys, hasCustomContribution);
+  const hasCustomContribution = !get(collective, 'settings.disableCustomContributions', true);
```

Comment:
I don't think the default is `true`, should be `false` right?

Reply:
Ops, yes that's correct. The custom contributions needs to be enabled by default so that makes sense; whereas the crypto contributions are disabled by default. I've changed this. 👍🏽